packages/console/src/components/CopyToClipboard/index.tsx

```diff
@@ -12,14 +14,24 @@ type Props = {
   value: string;
   className?: string;
   variant?: 'text' | 'contained' | 'border' | 'icon';
+  isContentHidden?: boolean;
```

Comment:
`isContentHidden` is actually the flag to tell the component if the content visibility can be toggled, not to hide the content... So it's a bit confusing IMO.
Any better name? How about `hasVisibilityToggle`?

Reply:
updated